apps/frontend/src/components/CourseBrowser/Filters/index.tsx

```diff
@@ -283,6 +283,39 @@ export default function Filters({
             </div>
           );
         })}
+        <p className={styles.label}>Level</p>
+        {Object.values(Level).map((level) => {
+          const active = currentLevels.includes(level as Level);
```

Comment:
This file will probably require more work to transfer over university/l&s requirements here too